dtfs-central-api/api-tests/v1/utilisation-reports/put-fee-record-correction-transient-form-data.api-test.ts

```diff
@@ -0,0 +1,188 @@
+import { HttpStatusCode } from 'axios';
+import {
+  FEE_RECORD_STATUS,
+  FeeRecordEntityMockBuilder,
+  MAX_RECORD_CORRECTION_ADDITIONAL_INFO_CHARACTER_COUNT,
+  RECONCILIATION_IN_PROGRESS,
+  RECORD_CORRECTION_REASON,
+  RecordCorrectionTransientFormData,
+  UtilisationReportEntityMockBuilder,
+} from '@ukef/dtfs2-common';
+import { withSqlIdPathParameterValidationTests } from '@ukef/dtfs2-common/test-cases-backend';
+import { testApi } from '../../test-api';
+import { SqlDbHelper } from '../../sql-db-helper';
+import { aTfmSessionUser } from '../../../test-helpers';
+import { PutFeeRecordCorrectionTransientFormDataPayload } from '../../../src/v1/routes/middleware/payload-validation';
+
+console.error = jest.fn();
+
+const BASE_URL = '/v1/utilisation-reports/:reportId/fee-records/:feeRecordId/correction-transient-form-data';
+
+describe(`PUT ${BASE_URL}`, () => {
+  const getUrl = (reportId: number | string, feeRecordId: number | string) =>
+    BASE_URL.replace(':reportId', reportId.toString()).replace(':feeRecordId', feeRecordId.toString());
```

Comment:
this will be a little easier to read if split up, as in const replaced... = reportId.toString()).replace(':feeRecordId', feeRecordId.toString();
then BASE_URL.replace uses that const

Reply:
Sorry not following this exactly, do you mean you'd prefer the following?
```
const replacedReportId = BASE_URL.replace(':reportId', reportId.toString());
return replacedReportId.replace(':feeRecordId', feeRecordId.toString());
```

Reply:
yes something easier to read like that